### ISSUE TITLE
Form fixed text does not work with horizontal form

### ISSUE THREAD

**Issue by dsharrison** [salesforce-ux/design-system#410]
When using fixed form text, setting the form to a horizontal display mode causes the fixed text to be pushed to a new line instead of appending to the front or back of the input.

![form-horizontal-fixed-text](https://cloud.githubusercontent.com/assets/9137905/23522663/ae1e4f28-ff38-11e6-9ae5-b3e21b542601.png)

```html
<h1 class="slds-text-heading--large slds-m-bottom--small slds-m-top--large">Standard Form</h1>
<div class="slds-form">
  <div class="slds-form-element">
    <label class="slds-form-element__label" for="text-input-01">Input Label</label>
    <div class="slds-form-element__control slds-input-has-fixed-addon">
      <span class="slds-form-element__addon">$</span>
      <input type="text" id="text-input-01" class="slds-input" placeholder="Placeholder Text" />
      <span class="slds-form-element__addon">%</span>
    </div>
  </div>
</div>

<h1 class="slds-text-heading--large slds-m-bottom--small slds-m-top--large">Horizontal Form</h1>
<div class="slds-form--horizontal">
  <div class="slds-form-element">
    <label class="slds-form-element__label" for="text-input-01">Input Label</label>
    <div class="slds-form-element__control slds-input-has-fixed-addon">
      <span class="slds-form-element__addon">$</span>
      <input type="text" id="text-input-01" class="slds-input" placeholder="Placeholder Text" />
      <span class="slds-form-element__addon">%</span>
    </div>
  </div>
</div>

```

**Comment by kaelig** [comment]
Thank you for the bug report, we'll look into it!

**Comment by dsharrison** [comment]
I got a note this is being automatically closed, was the issue addressed?

**Comment by engai** [comment]
Hi @dsharrison, Stale bot was turned on to have users explicitly indicate certain issues that they are still experiencing and need addressed. This is a part of the larger effort to improve our processes around Github issues.

We'll looking at this as a team and get back to you shortly.

**Comment by engai** [comment]
@dsharrison Looks like the display: flex property of `.slds-input-has-fixed-addon` is being unintentionally overridden when its within a `.slds-form_horizontal`.

I've filed a bug on our end for it and will let you know when we commit to and complete the work.

Thanks for bringing this to our attention.

**Comment by ayeshakmaz** [comment]
Thanks for catching this @dsharrison, a fix has been added and will be available with the Winter '19 release.

**Comment by dsharrison** [comment]
Thanks for the update!

**Comment by istiti** [comment]
@engai does this fixed in 2.7.1 ? because it seems working horizontally but one thing is missing, padding:left must go over label and it not 
![image](https://user-images.githubusercontent.com/2789203/46000074-e04cc880-c0a7-11e8-9cf3-cb8495d74ebc.png)

while it must be 
![image](https://user-images.githubusercontent.com/2789203/46000109-f8bce300-c0a7-11e8-82e8-5574a0ff9c0a.png)
